### PR TITLE
prepare AzureMonitorExporter and AzureMonitorDistro for release

### PR DESCRIPTION
**Pull request by TimothyMothra**

### CODE REVIEW

File: sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/CHANGELOG.md [hunk 1]
```diff
@@ -1,16 +1,12 @@
 # Release History
 
-## 1.0.0-beta.5 (Unreleased)
+## 1.0.0-beta.5 (2023-07-11)
```
Comment (rajkumar-rangaraj) [inline sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/CHANGELOG.md:3]: Azure Source addition update needed here.

Comment (rajkumar-rangaraj) [inline sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/CHANGELOG.md:3]: 2023-07-11 => 2023-07-13

Comment (TimothyMothra) [reply sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/CHANGELOG.md:3]: done.

File: sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md [hunk 2]
```diff
@@ -1,18 +1,22 @@
 # Release History
 
-## 1.0.0-beta.13 (Unreleased)
+## 1.0.0-beta.13 (2023-07-13)
 
 ### Features Added
 
+* Added Azure.* activity sources.
```
Comment (rajkumar-rangaraj) [inline sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md:7]: Move this section to distro.

Comment (TimothyMothra) [reply sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md:7]: sorry, done

File: sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md [hunk 3]
```diff
@@ -1,18 +1,19 @@
 # Release History
 
-## 1.0.0-beta.13 (Unreleased)
+## 1.0.0-beta.13 (2023-07-13)
 
 ### Features Added
 
+
```
Comment (rajkumar-rangaraj) [inline sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md:7]: nit: remove extra line here.

**Review by rajkumar-rangaraj — APPROVED** [review]

**Pull request merged**